networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/PeerManager.java

```diff
@@ -58,6 +59,9 @@ public class PeerManager implements ConnectionHandler {
   private final Subscribers<PeerConnectedSubscriber<Peer>> connectSubscribers =
       Subscribers.create(true);
 
+  private final AtomicLong inboundPeers = new AtomicLong(0);
+  private final AtomicLong outboundPeers = new AtomicLong(0);
```

Comment:
this is likely a lot simpler to just filter on the map rather than add new counters...